src/ansible_navigator/action_runner.py

```diff
@@ -1,4 +1,16 @@
-"""Jump to one action."""
+"""A mechanism to start the main application loop for the text user interface.
+
+Invoked after settings are parsed, a loop is entered and only
```

Comment:
Modules are not invoked but imported, class instances are initialized and their methods are invoked. So this could probably go into some of those places it tries to refer to?

Reply:
Or at least explicitly mention what it is that is invoked if this is meant to be at the beginning of the document.

Reply:
this is why knowing something about the code works against me.... One file, one class, named the same, it is only implied that that comment refers the sole class in the file. updating, thx